flexmeasures/data/models/planning/utils.py

```diff
@@ -78,20 +77,14 @@ def ensure_storage_specs(
     if specs is None:
         specs = {}
 
-    sensor: Optional[Sensor] = None
```

Comment:
Instead of calling `ensure_storage_specs` with `sensor_id=sensor.id` and then querying for the corresponding Sensor, can't we just call it with `sensor=sensor`?

Reply:
I checked and you are correct, we indeed always have the object already.